src/index.html

```diff
@@ -22,6 +22,398 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__logo"
+      >
+        <img
+          class="header__logo-img"
+          src="images/logo.png"
+          alt="moyo_logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list-item">
+            <a
+              href="#apple"
+              class="nav__list-item-link is-active"
```

Comment:
Fix classes everywhere
```suggestion
          <li class="nav__item">
            <a
              href="#apple"
              class="nav__link is-active"
```